packages/framer-motion/src/render/utils/animation-state.ts

```diff
@@ -212,8 +214,9 @@ export function createAnimationState(
              * Build an object of all the resolved values. We'll use this in the subsequent
              * animateChanges calls to determine whether a value has changed.
              */
+            // TODO Resolve with options
```

Comment:
Does this TODO need addressing before merge? Or is it no longer relevant?

Reply:
Removed, I think this was an idle thought